lib/ansible/modules/windows/win_domain_membership.ps1

```diff
@@ -128,7 +129,11 @@ Function Join-Domain {
     $argstr = $add_args | Out-String
     Write-DebugLog "calling Add-Computer with args: $argstr"
     try {
-        $add_result = Add-Computer @add_args
+        if(Get-ADObject $new_hostname -eq $null -or (Get-ADObject $new_hostname -ne $null -and $force_replace_host)) {
+            $add_result = Add-Computer @add_args
+        } else {
+            Fail-Json -obj $result -message "failed to join domain: hostname already exists in AD and `force_replace_host` isn't `true`"
```

Comment:
Looks like last ` backtick is an escape character and didn't allow the close out of your message string.

Reply:
Weird, the failure is on a completely different part of file, but this seems like a valid change nontheless.